.browserslistrc

```diff
@@ -0,0 +1 @@
+extends @cerner/browserslist-config-terra
```

Comment:
I'm pretty sure this will need to be ignored or we will run into the same issue that we are currently having.

Reply:
npm ignored right?

Reply:
Added here https://github.com/cerner/terra-application/pull/114/commits/042038071dfb6be41c14f2f291bebc18d16503c9.

Reply:
Looks like it's going to be ignored by this: https://github.com/cerner/terra-application/blob/main/package.json#L65-L71. Let's make sure this file doesn't get packed up when you run npm pack but I think we should be good without the npm ignore file.